### PR TITLE
chore: update topic heading as per locked status

### PR DESCRIPTION
**Pull request by anshumanv**
Ref - https://github.com/webpack/webpack.js.org/issues/3446

### CODE REVIEW

File: src/components/topic/topic.jsx [hunk 1]
```diff
@@ -8,181 +8,195 @@ import './topic-style';
...
-            .onChangeSettings(topic.id, { title, description })
-            .then(success => {
-                if (success) {
-                    this.setState({
-                        editing: false
-                    });
-                }
-            });
-    }
+	constructor(props) {
```
Comment (montogeek) [inline src/components/topic/topic.jsx]: Please revert formatting changes, it is unclear what was changed.

Comment (anshumanv) [reply src/components/topic/topic.jsx]: Done, PTAL. 👍

File: src/components/topic/topic.jsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default class Topic extends React.Component {
 
                 <section className={ `${block}__vote` }>
                     <div className={ `${block}__title ${block}__title--vote` }>
-                        Place Your Vote
+                        {topic.locked ? 'Voting Ended' : 'Place Your Vote'}
```
Comment (EugeneHlushko) [inline src/components/topic/topic.jsx]: Did you mean `disabled`? By the look of it you are checking if its locked 
```suggestion
                        {topic.locked ? 'Voting not allowed' : 'Place Your Vote'}
```

Comment (anshumanv) [reply src/components/topic/topic.jsx]: Done

File: src/components/topic/topic.jsx [hunk 3]
```diff
@@ -76,7 +76,7 @@ export default class Topic extends React.Component {
 
                 <section className={ `${block}__vote` }>
                     <div className={ `${block}__title ${block}__title--vote` }>
-                        Place Your Vote
+                        {topic.locked ? 'Voting not allowed' : 'Place your vote'}
```
Comment (EugeneHlushko) [inline src/components/topic/topic.jsx]: Im sorry, grammar is off:
```suggestion
                        {topic.locked ? 'Voting is not allowed' : 'Place your vote'}
```

Comment (anshumanv) [reply src/components/topic/topic.jsx]: Fixed.

**Comment by EugeneHlushko** [comment]
Thanks!

**Comment by anshumanv** [comment]
Cool, thanks for the review!

**Pull request merged**